src/ado2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -161,16 +167,18 @@ public async Task Invoke(string githubOrg, string adoOrg, FileInfo output, bool
                     repos.Add(org, new Dictionary<string, IEnumerable<string>>());
 
                     var teamProjects = await ado.GetTeamProjects(org);
-
-                    foreach (var teamProject in teamProjects)
+                    if (string.IsNullOrEmpty(adoTeamProject))
                     {
-                        _log.LogInformation($"  Team Project: {teamProject}");
-                        var projectRepos = await ado.GetRepos(org, teamProject);
-                        repos[org].Add(teamProject, projectRepos);
-
-                        foreach (var repo in projectRepos)
+                        foreach (var teamProject in teamProjects)
+                        {
+                            await GetTeamProjectRepos(ado, repos, org, teamProject);
+                        }
+                    }
+                    else
+                    {
+                        if (teamProjects.Any(o => o.Equals(adoTeamProject, StringComparison.OrdinalIgnoreCase)))
                         {
-                            _log.LogInformation($"    Repo: {repo}");
+                            await GetTeamProjectRepos(ado, repos, org, adoTeamProject);
                         }
```

Comment:
```suggestion
                            await GetTeamProjectRepos(ado, repos, org, adoTeamProject);
                        }
                        else
                        {
                            _log.LogWarning($"ADO Team Project provided cannot be found [{adoTeamProject}]");
                        }
```

Reply:
I thought about logging a warning but if you have 3 orgs that it's looping over and the team project exists in 1 of them then you'd get 2 warnings thinking something is wrong but it worked as expected

Reply:
What I'll do is capture if a team project was found and outside the org loop log the warning, same same, just a little more tracking code :)